src/Maestro/DependencyUpdateErrorProcessor/Program.cs

```diff
@@ -48,18 +50,21 @@ public static void Configure(IServiceCollection services)
             services.Configure<DependencyUpdateErrorProcessorOptions>(
                 (options, provider) =>
                 {
-                    var config = provider.GetRequiredService<IConfiguration>();
-                    if (!bool.TryParse(config["EnableDependencyUpdateErrorProcessor"], out var errorProcessorFlag))
+                    var config1 = provider.GetRequiredService<IConfiguration>();
```

Comment:
why the `1`?

Reply:
ReSharper did it... Need to fix them all..